src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerAssemblyLoader.Core.cs

```diff
@@ -83,6 +85,60 @@ private partial Assembly Load(AssemblyName assemblyName, string assemblyOriginal
             return loadContext.LoadFromAssemblyName(assemblyName);
         }
 
+        /// <summary>
+        /// Is this a registered analyzer file path that the loader knows about.
+        /// 
+        /// Note: this is using real paths, not the original file paths
```

Comment:
not sure the distinction between "real path" and "original file path".  does this mean "the file path after shadow copying has happened"?

Reply:
Essentially _real_ means the result after `IAnalyzerPathResolver` ran and _original_ is before. The terminology is used in a number of places in this type when the distniction is important.

Reply:
commentd later.  but `original/final` reads a lot better/clearer for me.  however, as long as we doc what 'real' means somewhere, and use it consistently, i'm fine with that term.